cmd/dep/status.go

```diff
@@ -369,12 +369,10 @@ func runStatusAll(out outputter, p *dep.Project, sm *gps.SourceMgr) error {
 	rm, _ := ptree.ToReachMap(true, true, false, nil)
 
 	external := rm.Flatten(false)
-	roots := make(map[gps.ProjectRoot][]string)
-	var errs []string
+	roots := make(map[gps.ProjectRoot][]string, len(external))
```

Comment:
We know the maximum required capacity for `roots` is the number of external imports, so we should specify the capacity here.